trunk/SUAVE/Methods/Noise/Fidelity_One/Noise_Tools/print_airframe_output.py

```diff
@@ -0,0 +1,111 @@
+## @ingroupMethods-Noise-Fidelity_One-Noise_Tools
+# print_airframe_output.py
+# 
+# Created:  Oct 2020, M. Clarke
+
+# ----------------------------------------------------------------------        
+#   Imports
+# ----------------------------------------------------------------------    
+
+import numpy as np
+from SUAVE.Core            import Units  
+
+## @ingroupMethods-Noise-Fidelity_One-Noise_Tools
+def print_airframe_output(tag, filename,velocity,nsteps,time,altitude,M,angle,distance_vector,PNLT_wing,phi,
```

Comment:
this is a lot of inputs; maybe group some into different dictionaries to pass in then unpack inside?

Reply:
Done